Core/src/main/java/com/plotsquared/core/command/DebugExec.java

```diff
@@ -89,12 +89,12 @@ public DebugExec() {
 /*
         try {
             if (PlotSquared.get() != null) {
-                File file = new File(PlotSquared.get().IMP.getDirectory(),
+                File file = new File(PlotSquared.imp().getDirectory(),
                     Settings.Paths.SCRIPTS + File.separator + "start.js");
                 if (file.exists()) {
                     init();
                     String script = StringMan.join(Files.readLines(new File(new File(
-                            PlotSquared.get().IMP.getDirectory() + File.separator
+                            PlotSquared.imp().getDirectory() + File.separator
```

Comment:
If we're changing code that's been commented out, should it be fixed properly?

Reply:
Search and replace ^-^ But yes, probably. Although we should probably look over scripting in general. The scripting engine has been deprecated and most scripts are just broken.

Reply:
You should always do the best job you can with the tools, resources, skill and knowledge you have.  
Just because it's commented out don't mean it can't be used later, and if it's done properly, at the time it's touched, then it's more likely it can be used without pain.  That's not to say that other code won't change nullifying the changes, or that section all together, but if you're going to touch it, touch it like you're about to make love to it!